content/en/blog/_posts/2022-08-16-PSP-historical-context/index.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: blog
+title: "PodSecurityPolicy: The Historical Context"
+date: 2022-08-16
+slug: podsecuritypolicy-the-historical-context
+---
+
+**Author:** Mahé Tardy (Quarkslab)
+
+The PodSecurityPolicy (PSP) admission controller will be removed as of
+Kubernetes v1.25. Its deprecation was announced and detailed in the blog post
+[PodSecurityPolicy Deprecation: Past, Present, and Future](/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/),
+published for the Kubernetes v1.21 release.
+
+This article aims at providing historical context on the birth and evolution of
+PSP, to explain why the feature never made it to stable and had to be removed
+in favor of a new in-tree admission controller. Let's recap what
+PodSecurityPolicy is, why it was necessary, the historical context of its
+addition to Kubernetes, and finally, why it was removed and replaced by Pod
+Security admission control.
```

Comment:
```suggestion
This article aims to provide historical context on the birth and evolution of PodSecurityPolicy
(PSP), explain why the feature never made it to stable, and show why it was removed and replaced by Pod
Security admission control.
```
Are lines 18-20 redundant? If so, how about this proposed revision?

Reply:
Yes correct, these things were written separately and now that I put them just asides it's redundant!

It should be the "motivation" to read the article.